dbt_project.yml

```diff
@@ -1,5 +1,5 @@
 name: 'jira'
-version: '0.1.4'
+version: '0.2.0'
```

Comment:
Although not needed, we should probably update the integration_tests `dbt_project.yml` version to v0.2.0 for consistency.